app/objects/c_operation.py

```diff
@@ -41,12 +41,11 @@ class OperationSchema(ma.Schema):
     objective = ma.fields.Nested(ObjectiveSchema())
     use_learning_parsers = ma.fields.Boolean()
 
```

Comment:
We may also need to add `source` (fact source) and `group` to the schema to allow users to specify group name and which fact source they want to use for the operation